### PR TITLE
MEN-4998: monitoring tests adjustmets

### PR DESCRIPTION
**Pull request by 0lmi**
https://tracker.mender.io/browse/MEN-4998

### CODE REVIEW

File: tests/tests/test_monitor_client.py [hunk 1]
```diff
@@ -308,17 +308,17 @@ def test_monitorclient_alert_email(self, monitor_commercial_setup_no_client):
         for m in messages:
             logger.debug("got message:")
             logger.debug("             body: %s", m.get_body().get_content())
-            logger.debug("             To: %s", m["To"])
+            logger.debug("             To: %s", m["Bcc"])
```
Comment (lluiscampos) [inline tests/tests/test_monitor_client.py:311]:
```suggestion
            logger.debug("             Bcc: %s", m["Bcc"])
```

Comment (lluiscampos) [reply tests/tests/test_monitor_client.py:311]: (same suggestion for all the ones below)

File: tests/tests/test_monitor_client.py [hunk 2]
```diff
@@ -308,17 +308,17 @@ def test_monitorclient_alert_email(self, monitor_commercial_setup_no_client):
...
 
         messages_count = len(messages)
         assert messages_count > 1
         m = messages[1]
-        assert "To" in m
+        assert "Bcc" in m
         assert "From" in m
         assert "Subject" in m
-        assert m["To"] == user_name
+        assert m["Bcc"] == user_name
```
Comment (lluiscampos) [inline tests/tests/test_monitor_client.py:321]: Will this work? Won't `m["Bcc"]` be an array?

Comment (0lmi) [reply tests/tests/test_monitor_client.py:321]: it seems like it works (related [job](https://storage.googleapis.com/gitlab-gprd-artifacts/73/87/738722f98bfc927b70d946e82fd4e07a8c8ace4998154d3998220c339ad19dcc/2021_09_14/1588875566/1713666092/job.log?response-content-type=text%2Fplain%3B%20charset%3Dutf-8&response-content-disposition=inline&GoogleAccessId=gitlab-object-storage-prd@gitlab-production.iam.gserviceaccount.com&Signature=d9tDnvRvmx%2BtBtqfn7Hi8NyNk%2BdhFVYUjY7gn0ncKTbHgUwerqMjQPnGqFFp%0A8KkKwaAC9hjPrHR5EFbQqMm174i92Dq4Yg6s6hErTScE7du4Sp0S63Ga9Ijr%0ALM4JjtR69Plkk9Zulk0s0dCv3qmVYWvlg6q3tpJDP%2F%2FqCaNNUwPZb2bvqbo%2B%0A1mxS1wxDzubLNf2ADsX0UeBVQphuCLJSG9%2Bxul6IU1eIflIFqHN9wz%2Fk%2FMVw%0A98pnIpY3bV3%2F4t8j4Z4M02KXxQ%2BCsg4vfF4tGkWS3Ur20Ak3r%2B0sBiPPTfi7%0AZyOhRrFL0U6RHXKRtN7ob5lrzMF2MQFl0v1cmWUQTA%3D%3D&Expires=1631630276))

Comment (lluiscampos) [reply tests/tests/test_monitor_client.py:321]: That output link has expired, though. Can you link to the pipeline job?

Comment (0lmi) [reply tests/tests/test_monitor_client.py:321]: https://gitlab.com/Northern.tech/Mender/mender-qa/-/jobs/1588875566

Comment (0lmi) [reply tests/tests/test_monitor_client.py:321]: It's used in all 6 tests and the failure is not caused by this:
PASSED test_monitorclient_alert_email
PASSED test_dbus_subsystem
PASSED test_monitorclient_alert_store
PASSED test_dbus_pattern_match
FAILED  test_monitorclient_alert_email_rbac
PASSED test_dbus_bus_filter

**Review by lluiscampos — APPROVED** [review]

**Comment by 0lmi** [comment]
https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/371168854

**Comment by 0lmi** [comment]
https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/371242624

**Comment by 0lmi** [comment]
https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/371345693

**Review by merlin-northern — APPROVED** [review]

**Pull request merged**